manifests/kots-config.yaml

```
@@ -3,297 +3,703 @@ apiVersion: kots.io/v1beta1
+        help_text: |-
+          Set your TomTom LDS Api Key
+        required: true
+        type: text
+      - name: locationDataServicesGoogleCredentials
+        title: LDS Google credentials
+        when: '{{repl and (ConfigOptionEquals "locationDataServicesGeocodingProvider" "geocoding_google") (ConfigOptionEquals "locationDataServicesEnable" "1") }}'
+        help_text: |-
+          Set your Google LDS Api Key
+        required: true
+        type: text
+      - name: locationDataServicesMapboxCredentials
+        title: LDS Mapbox credentials
+        when: '{{repl and (or (ConfigOptionEquals "locationDataServicesGeocodingProvider" "geocoding_mapbox") (ConfigOptionEquals "locationDataServicesIsolinesProvider" "isolines_mapbox")) (ConfigOptionEquals "locationDataServicesEnable" "1") }}'
+        help_text: |-
+          Set your Mapbox LDS Api Key
+        required: true
+        type: text
+      - name: locationDataServicesHereCredentials
+        title: LDS Here credentials
+        when: '{{repl and (or (ConfigOptionEquals "locationDataServicesGeocodingProvider" "geocoding_here") (ConfigOptionEquals "locationDataServicesIsolinesProvider" "isolines_here") (ConfigOptionEquals "locationDataServicesRoutingProvider" "routing_here")) (ConfigOptionEquals "locationDataServicesEnable" "1") }}'
+        help_text: |-
+          Set your Here LDS Api Key
+        required: true
+        type: text
+      - name: locationDataServicesTraveltimeAppId
+        title: LDS Traveltime App Id
+        when: '{{repl and (ConfigOptionEquals "locationDataServicesIsolinesProvider" "isolines_traveltime") (ConfigOptionEquals "locationDataServicesEnable" "1") }}'
+        help_text: |-
+          Set your Traveltime App Id
+        required: true
+        type: text
+      - name: locationDataServicesTraveltimeApiKey
+        title: LDS Traveltime Api key
+        when: '{{repl and (ConfigOptionEquals "locationDataServicesIsolinesProvider" "isolines_traveltime") (ConfigOptionEquals "locationDataServicesEnable" "1") }}'
+        help_text: |-
+          Set your Traveltime Api Key
+        required: true
+        type: text
+
+      ## CDW
+      - name: enableCDW
+        title: CARTO Data Warehouse
+        help_text: >-
+          The CARTO Data Warehouse is a default connection provided to every CARTO organization,
+          designed to help users get started with the platform.
+          It grants access to resources such as cloud storage and cloud computing provided by CARTO,
+          including access to demo datasets and spatial datasets from CARTO’s Data Observatory.
+          It uses the same structure as a Google BigQuery connection, with projects, datasets, and tables, and includes
+          private datasets for individual users and shared datasets accessible to all users within an organization.
+        type: bool
+        default: "1"
```

```suggestion
        default: "0"
```
Unfortunatly the default should be disabled 😢